browser/src/control/Control.UIManager.js

```diff
@@ -1022,21 +1022,23 @@ L.Control.UIManager = L.Control.extend({
 	/// Shows tooltip in the document area
 	/// tooltipInfo contains rectangle (position in twips) and text properties
 	/// elem has to be jQuery selector output eg. $('.leaflet-layer')
-	showDocumentTooltip: function(tooltipInfo, elem) {
-		var split = tooltipInfo.rectangle.split(',');
-		var latlng = this.map._docLayer._twipsToLatLng(new L.Point(+split[0], +split[1]));
-		var pt = this.map.latLngToContainerPoint(latlng);
-
-		elem.tooltip();
-		elem.tooltip('enable');
-		elem.tooltip('option', 'content', tooltipInfo.text);
-		elem.tooltip('option', 'items', elem[0]);
-		elem.tooltip('option', 'position', { my: 'left bottom',  at: 'left+' + pt.x + ' top+' + pt.y, collision: 'fit fit' });
-		elem.tooltip('open');
-		document.addEventListener('mousemove', function() {
-			elem.tooltip('close');
-			elem.tooltip('disable');
-		}, {once: true});
+	showDocumentTooltip: function(tooltipInfo, elem, pos) {
+        var pt = this.map.latLngToContainerPoint(pos);
+        pt.y -=35; //Show tooltip above the cursor.
```

Comment:
35 is standard cell height? we will need to adjust that:
- with real cursor height
- with DPI of a user

(We can do it later, in the followup)

Reply:
It would be nice to fix it as followup.